### PR TITLE
[SIG-3565] Next incident button

### PR DESCRIPTION
**Pull request by MauritsR**
### Changes

- Adds button to final incident step to quickly navigate to a new incident form

### CODE REVIEW

File: src/signals/incident/components/form/HandlingMessage/__snapshots__/index.test.js.snap [hunk 1]
```diff
@@ -2,43 +2,35 @@
 
 exports[`Form component <HandlingMessage /> rendering should render empty correctly with no handling message 1`] = `
 <div
-  className="col-12 mode_input"
+  className="col-12"
```
Comment (aditudorache) [inline src/signals/incident/components/form/HandlingMessage/__snapshots__/index.test.js.snap:5]: Can we delete this snap and replace the tests with testing-library?

**Review by aditudorache — CHANGES_REQUESTED** [review]
Nice! If we can remove the old snapshot would be great!

File: src/signals/incident/components/form/HandlingMessage/__snapshots__/index.test.js.snap (hunk 1 above)
Comment (MauritsR) [reply src/signals/incident/components/form/HandlingMessage/__snapshots__/index.test.js.snap]: Fixed it :) now in TypeScript and without the snapshots (using testing-library)

**Review by aditudorache — APPROVED** [review]

**Comment by aditudorache** [comment]
Approved! Some tests still failing...

**Pull request merged**